### PR TITLE
Bookstore initialize project with components

### PR DESCRIPTION
**Pull request by X-Elie-X**
## In this PR I implemented the below:
- Installed linters.
- Installed   React App.
- Created file components.
- Add in the components of the file: Navbar.js, DisplayBooks.js, Categorys.js, books.js

### CODE REVIEW

File: src/components/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+const Bookslist = ({ books, handleDelete }) => (
+  <div className="booksList">
+    {books.map((book) => (
+      <div className="prevBooks" key={book.id}>
+        {book.title}
```
Comment (whiz25) [inline src/components/BooksList.js]: - Render a component of type `Book` here. A component allows for reusability of code across the app. Please make that change.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import Books from './components/Books';
+import Category from './components/Category';
+import './style.css';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
```
Comment (whiz25) [inline src/App.js:14]: - This should render the `BookList` component. Please change that.

File: src/components/BooksList.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        {book.author}
+        {book.category}
+        <button type="button" onClick={() => handleDelete(book.id)}>
```
Comment (whiz25) [inline src/components/BooksList.js]: - Please render a form to add a new book to the list of books. It shouldn't do anything at this stage.

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @X-Elie-X, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🎉 🎉 🎉 
- code is organized
- Some requirements have been met

## Required Changes ♻️
_Check the comments under the review._
- You haven't Implemented some required components:
   `Book`. This component specifies the properties of a book such as title
    `CreateNewBook`. This renders a form for creating a new book. 
- Rename the ` Category` component to `Categories`. It makes more sense that way.

# Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @X-Elie-X 🔥 

Good job on implementing the required changes. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**